1.8/overview/concepts.md

```diff
@@ -1,121 +1,282 @@
+- Tasks on DC/OS may opt-in to being placed on a specific virtual network and given a container-specific IP.
+- Virtual networks allow logical subdivision of the tasks running on DC/OS
+- Each task on a virtual network may be configured with optional address groups that virtually isolate communication to tasks on the same network and address group.
 
-# <a name="dockercontainerizer"></a>Docker Containerizer
+### <a name="node"></a>Node
 
-The Docker Containerizer enables launching docker containers using DC/OS.
+A DC/OS node is a virtual or physical machine on which a Mesos agent and/or Mesos master process runs. DC/OS nodes are networked together to form a DC/OS cluster.
 
-# <a name="datacenteroperatingsystem"></a>Datacenter operating system
+#### <a name="master-node"></a>Master Node
 
-A new class of operating system that spans all of the machines in a datacenter or cloud and organizes them to act as one big computer.
+A DC/OS master node is a virtual or physical machine that runs a collection of DC/OS components that work together to manage the rest of the cluster.
 
-# <a name="dcos"></a>DC/OS
+- Each master node contains (among other things) a Mesos master process.
+- Master nodes work in a quorum to provide consistency of cluster coordination. Three master nodes allows one to be down. Five master nodes allows two to be down (e.g. to allow for failure during a rolling update).
+- A cluster with only one master node is usable for development, but is not highly available and may not be able to recover from failure.
 
-The abbreviated form of the Datacenter Operating System.
+#### <a name="agent-node"></a>Agent Node
 
-# <a name="dcoscluster"></a>DC/OS Cluster
+A DC/OS agent node is a virtual or physical machine on which Mesos tasks are run.
 
-A group of Mesos master and agent nodes.
+- Each agent node contains (among other things) a Mesos agent process.
+- Agent nodes may be [private](#private-agent-node) or [public](#public-agent-node), depending on agent and network configuration.
+- For more information, see [Network Security](/docs/1.8/administration/securing-your-cluster/) and [Adding Agent Nodes](/docs/1.8/administration/installing/custom/add-a-node/).
 
-# <a name="dcosmarathon"></a>DC/OS Marathon
+##### <a name="private-agent-node"></a>Private Agent Node
 
-The native Marathon instance that is the “init system” for DC/OS. It starts and monitors DC/OS applications and services.
+A private agent node is an agent node that is on a network that** ***does not* have ingress access from outside of the cluster via the cluster’s infrastructure networking.
 
-# <a name="dcosservice"></a>DC/OS service
+- The Mesos agent on each private agent node is (by default) configured with none of its resources allocated to any specific Mesos roles (`*`).
+- Most service packages install by default on private agent nodes.
+- Clusters are usually made up of mostly private agent nodes.
```

Comment:
`Clusters are usually made up of mostly private agent nodes.` to `Clusters are generally comprised of mostly private agent nodes.`